video_creation/final_video.py

```diff
@@ -436,9 +436,11 @@ def on_update_example(progress) -> None:
 
         old_percentage = pbar.n
         pbar.update(100 - old_percentage)
+
     pbar.close()
     save_data(subreddit, filename + ".mp4", title, idx, background_config["video"][2])
+
     print_step("Removing temporary files 🗑")
-    cleanups = cleanup(reddit_id)
-    print_substep(f"Removed {cleanups} temporary files 🗑")
```

Comment:
why remove the info print

Reply:
cleanup util is hardcoded to return the int 1. In the latest state, the print statement makes no sense for context.

It should either be like this, where it doesn't return how many temporary files are deleted, or the cleanup util should be added on to return the number of files it returns instead of it being hard coded to 1.